module/userProjects/helper.js

```diff
@@ -1293,6 +1307,21 @@ module.exports = class UserProjectsHelper {
             } else {
                 projectDetails.data.status = UTILS.convertProjectStatus(projectDetails.data.status);
             }
+            // make templateUrl downloadable befor passing to front-end
+            if ( projectDetails.data.certificate &&
+                 projectDetails.data.certificate.templateUrl &&
+                 projectDetails.data.certificate.templateUrl !== "" 
+            ) {
+                let certificateTemplateDownloadableUrl =
+                    await coreService.getDownloadableUrl(
+                        {
+                            filePaths: [projectDetails.data.certificate.templateUrl]
+                        }
+                    );
+                    if ( certificateTemplateDownloadableUrl.success ) {
+                        projectDetails.data.certificate.templateUrl = certificateTemplateDownloadableUrl.data[0].url;
```

Comment:
@VISHNUDAS-tunerlabs in MongoDB anywhere we should not store complete URL path.

Reply:
@aks30 we are not storing complete url.  We are creating complete url before we send them to front-end